packages/flutter/lib/src/services/raw_keyboard.dart

```diff
@@ -333,7 +341,7 @@ abstract class RawKeyEvent with Diagnosticable {
     final String type = message['type'] as String;
     switch (type) {
       case 'keydown':
-        return RawKeyDownEvent(data: data, character: message['character'] as String);
+        return RawKeyDownEvent(data: data, character: message['character'] as String); // Missing for Mac/iOS?
```

Comment:
Hmm. I actually think this is missing for all platforms except Android. Nice find: I can't believe I didn't see that earlier.

Don't bother fixing it here, this PR is big enough as it is. I captured this in [this issue](https://github.com/flutter/flutter/issues/65401).

(do remove the comment though)